src/EventServiceProvider.php

```diff
@@ -112,10 +110,22 @@ protected function createEventBridgeDriver(array $config): \Illuminate\Contracts
      */
     public static function prepareConfigurationCredentials(array $config): array
     {
-        if (Arr::has($config, ['key', 'secret'])) {
+        if (static::configHasCredentials($config)) {
             $config['credentials'] = Arr::only($config, ['key', 'secret', 'token']);
         }
 
         return $config;
     }
+
+    /**
+     * Make sure some AWS credentials were provided to the configuration array.
+     *
+     * @return bool
```

Comment:
In this JavaDoc string, can you type the input argument as well?

Reply:
It's PHPDoc, which I guess is pretty much the same. We could indeed add `@param  array  $config` - for consistency, but believe we just took the code that was added as part of the [PR](https://github.com/Pod-Point/laravel-aws-pubsub/pull/64/files#diff-010b2120a082db1cebaf7d5ffa6e2d259fdad880cb9b6f350a262026cf2ffd15) to bring this functionality into the `v1` branch. 

Could we also bring in over the EventBridgeBroadcasterTest and SnsBroadcasterTest?